website/docs/en/guide/start/npm-packages.mdx

```diff
@@ -1 +1,30 @@
 # Packages
+
+This document showcases all the npm package information maintained by Rslib team.
+
+## @rslib/core
+
+![](https://img.shields.io/npm/v/@rslib/core?style=flat-square&colorA=564341&colorB=F8F5FF)
+
+Rslib core package that provides CLI commands and build capabilities based on Rsbuild.
+
+- [npm](https://npmjs.com/package/@rslib/core)
+- [Source Code](https://github.com/web-infra-dev/rslib/tree/main/packages/core)
+
+## rsbuild-plugin-dts
+
+![](https://img.shields.io/npm/v/rsbuild-plugin-dts?style=flat-square&colorA=564341&colorB=F8F5FF)
+
+Rsbuild plugin that provides support for emitting declaration files for Typescript in Rslib.
+
+- [npm](https://npmjs.com/package/rsbuild-plugin-dts)
+- [Source Code](https://github.com/web-infra-dev/rslib/tree/main/packages/plugin-dts)
+
+## create-rslib
+
+![](https://img.shields.io/npm/v/create-rslib?style=flat-square&colorA=564341&colorB=F8F5FF)
+
+Used to create a new Rslib project.
```

Comment:
Scaffolding for Rslib projects.

Reply:
I just follow Rsbuild docs about this package.